python/fastexcel/__init__.py

```diff
@@ -178,6 +182,7 @@ def load_sheet(
         skip_rows: int = 0,
         n_rows: int | None = None,
         schema_sample_rows: int | None = 1_000,
+        use_columns: list[str] | list[int] | None = None,
```

Comment:
I reckon it's important to support `str` syntax like `"A:E”` or `“A,C,E:F"`
IMO way easier for 90% of cases

Reply:
Hmm why not, but maybe in a second PR ? This one is already quite big :grimacing: If we could validate the types and logic with this one, I'll add support for a new variant in another PR

Reply:
Ok